packages/apps/desktop-viewer-test/src/frontend/components/signin/SignIn.tsx

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import "./SignIn.scss";
+
+import { IModelApp } from "@bentley/imodeljs-frontend";
+import { Button } from "@itwin/itwinui-react";
+import React, { useState } from "react";
+
+export const SignIn = () => {
+  const [signingIn, setSigningIn] = useState(false);
+
+  const onSignInClick = () => {
+    setSigningIn(true);
+    IModelApp.authorizationClient?.signIn();
+  };
+
+  return (
+    <div className="signin-container">
```

Comment:
instead of this signin container, you could use <FillCentered /> from ui-core

Reply:
prefer to not use ui-core, especially something this simple